lib/matplotlib/transforms.py

```diff
@@ -115,7 +115,7 @@ def __getstate__(self):
     def __setstate__(self, data_dict):
         self.__dict__ = data_dict
         # turn the normal dictionary back into a WeakValueDictionary
-        self._parents = WeakValueDictionary(self._parents)
+        self._parents = WeakValueDictionary(self.__dict__['_parents'])
```

Comment:
The weak reference is destroyed during pickling, thus self._parents does not exist anymore at this point. __getstate__ however saved it in data_dict['_parents'] from which it is now restored.